src/Stack/Docker.hs

```diff
@@ -639,7 +639,8 @@ inspects :: (HasProcessContext env, HasLogFunc env)
          => [String] -> RIO env (Map String Inspect)
 inspects [] = return Map.empty
 inspects images =
-  do maybeInspectOut <- try (readDockerProcess ("inspect" : images))
+  do maybeInspectOut <-
+       try (BL.toStrict . fst <$> proc "docker" ("inspect" : images) readProcess_)
```

Comment:
Is it worth fixing `readDockerProcess` rather than overriding it? I notice that there's a FIXME in the below code.

https://github.com/commercialhaskell/stack/blob/b01c526c733d30ab2df8fa296e0e02f6314c15be/src/Stack/Docker.hs#L844-L850

Reply:
That was what I tried first, but the current `readDockerProcess` behaviour of passing stderr straight through is desirable for some docker commands such as `docker pull` (so you can see the fancy pull progress that Docker outputs).

Reply:
If that's the case, I think the FIXME can be removed (and thanks for noticing that David).